docs/usage/getting-started/installing-onboarding.md

```diff
@@ -0,0 +1,133 @@
+# Installing and onboarding Renovate into repositories
+
+## Repository installation
+
+Renovate administrators can configure Renovate to either "autodiscover" installed repositories, or configure a fixed list of repository names to operate on.
+
+If the adminstrator has configured a fixed list of repositories then the only way to "install" Renovate on an additional repository is for it to be manually added for the next run or restart.
+
+Otherwise, the process for adding new repositories to a Renovate installation can vary:
+
+- Most commonly, you run Renovate as a dedicated "bot user" with global config option `autodiscover` set to `true`, meaning that it will run on every repository which it's been granted access to
+- If using a GitHub App (including WhiteSource Renovate on `github.com`) then you can install the app into a user or organization account and select either "All repositories", or "Select repositories" and pick them manually
+
+### Hosted GitHub.com App
+
+Installing/enabling WhiteSource's Renovate GitHub App is simple.
+
+First, navigate to [https://github.com/apps/renovate](https://github.com/apps/renovate) and click the Install button:
+
+![Github App Install button](assets/images/github-app-install.png)
```

Comment:
Our images don't show in the rich-preview on GitHub, will they they show on the published site after merge?

Reply:
I hope so. I haven't added any new images, just reused existing text